### PR TITLE
Fixes #21225: Move API doc create node in rudder

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/21225

### CODE REVIEW

**Comment by ElaadF** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/code_samples/curl/create-node/all.sh [hunk 1]
```diff
@@ -18,17 +18,26 @@ nodes.json:
...
-      "tags":[
-        "some",
-        "tags"
-      ],
-      "env":"prod",
-      "vars":{
-        "var1":"value1",
-        "var2":"value2"
+    "properties": [
+      {
```
Comment (ElaadF) [inline webapp/sources/api-doc/code_samples/curl/create-node/all.sh:22]: standardize properties format between `Get /nodes/{id}` and ` Put /nodes`

File: webapp/sources/api-doc/paths/nodes/all.yml [hunk 2]
```diff
@@ -52,3 +52,56 @@ get:
...
+                      type: string
+                      description: created nodes ID
+                      example: 378740d3-c4a9-4474-8485-478e7e52db52
+                  failed:
+                    type: array
+                    items:
+                      type: string
+                      description: failed nodes ID
+  tags:
+    - 🧩 Create Node
```
Comment (fanf) [inline webapp/sources/api-doc/paths/nodes/all.yml:103]: need to remove the plugin icon

File: webapp/sources/api-doc/paths/nodes/all.yml [hunk 3]
```diff
@@ -52,3 +52,56 @@ get:
     - lang: python
       source:
         $ref: ../../code_samples/python/nodes/all.py
+
+put:
+  summary: Create a new node
+  description: Create a new node
+  operationId: createNodes
```
Comment (fanf) [inline webapp/sources/api-doc/paths/nodes/all.yml:59]: should be singulare I think

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/all.yml [hunk 4]
```diff
@@ -52,3 +52,60 @@ get:
     - lang: python
       source:
         $ref: ../../code_samples/python/nodes/all.py
+
+put:
+  summary: Create a new node
```
Comment (fanf) [inline webapp/sources/api-doc/paths/nodes/all.yml:57]: Create one or several new nodes

File: webapp/sources/api-doc/paths/nodes/all.yml [hunk 5]
```diff
@@ -52,3 +52,60 @@ get:
...
+  description: Create a new node
```
Comment (fanf) [inline webapp/sources/api-doc/paths/nodes/all.yml:58]: Use the provided array of node information to create new nodes

**Review by fanf — APPROVED** [review]

**Pull request merged**